Author: Diane Trout <diane@ghic.org>
Description: Debian recently released xsdcxx 4.0 which included
 a parameter change to the C++ template for simple_type. This
 adds the required type as defined in one of the generated classes.
--- a/src/shared_conversions.h
+++ b/src/shared_conversions.h
@@ -29,7 +29,11 @@
 typedef boost::shared_ptr<cDateTime> cDateTimePtr;
 
 typedef ::xsd::cxx::tree::type type;
+#if XSD_INT_VERSION >= 400000L
+typedef ::xsd::cxx::tree::simple_type< char, type > simple_type;
+#else
 typedef ::xsd::cxx::tree::simple_type< type > simple_type;
+#endif
 typedef ::xsd::cxx::tree::date< char, simple_type > date;
 typedef ::xsd::cxx::tree::date_time< char, simple_type > date_time;
 
